src/ServiceControl.Audit.Persistence.Tests/AuditTests.cs

```diff
@@ -2,26 +2,32 @@
 {
     using System;
     using System.Collections.Generic;
-    using System.Text;
     using System.Threading.Tasks;
     using Auditing;
     using Infrastructure;
     using NServiceBus;
     using NUnit.Framework;
-    using ServiceControl.SagaAudit;
 
     [TestFixture]
     class AuditTests : PersistenceTestFixture
     {
         [Test]
         public async Task Basic_Roundtrip()
         {
-            var metadata = new Dictionary<string, object>();
+            var metadata = new Dictionary<string, object>
+            {
+                { "MessageIntent", MessageIntentEnum.Send },
+                { "CriticalTime", TimeSpan.FromSeconds(5) },
+                { "ProcessingTime", TimeSpan.FromSeconds(1) },
+                { "DeliveryTime", TimeSpan.FromSeconds(4) },
+                { "IsSystemMessage", false },
+                { "ContentLength", 25 }
```

Comment:
Note that all of these values are required for the RavenDb 5 implementation or it will not deserialize